index.html

```diff
@@ -37,6 +37,48 @@
     </nav>
 
 
+    <main>
+        <div class="theme">
+            <div class="flex">
+                <div>
+                    <h1 class="h1"> Meow in! </h1>
+                    <p> You know that feeling when you miss sunshine?
+                        Yep, <em> that </em> is how much we missed you. <strong> Welcome back</strong>.
+                    </p>
+                </div>
+                <img src="https://www.smashingmagazine.com/images/smashing-cat/singing-barista.svg" alt="logo"
+                    width="250" height="100" class="floatright" />
+            </div>
+            <form class="main-form">
+                <label class="block">
+                    <span class="margin-bottom"> Your email </span>
```

Comment:
The class `margin-bottom` is a little confusing because it doesn't say what the margin amount is.

Have a look here for for inspiration: https://tailwindcss.com/docs/margin. You see how the classes have a number in the name, that let's us know the value/amount of margin.